### ISSUE TITLE
Cannot complete tree at given position

### ISSUE THREAD

**Issue by indig** [polynote/polynote#29]
Seeing a lot of these errors in the logs:

```
[ForkJoinPool-1-worker-13] ERROR polynote.server.SocketSession - RuntimeException: Cannot complete tree at given position
polynote.kernel.RuntimeError: RuntimeException: Cannot complete tree at given position
	at polynote.server.SocketSession$$anonfun$toResponse$1$$anonfun$4.apply(SocketSession.scala:82)
	at polynote.server.SocketSession$$anonfun$toResponse$1$$anonfun$4.apply(SocketSession.scala:79)
...
Caused by: java.lang.RuntimeException: Cannot complete tree at given position
	at polynote.kernel.lang.scal.ScalaSource.polynote$kernel$lang$scal$ScalaSource$$completionResults$1(ScalaSource.scala:315)
	at polynote.kernel.lang.scal.ScalaSource$$anonfun$completionsAt$1$$anonfun$apply$28.apply(ScalaSource.scala:321)
	at polynote.kernel.lang.scal.ScalaSource$$anonfun$completionsAt$1$$anonfun$apply$28.apply(ScalaSource.scala:320)
	at cats.syntax.EitherOps$.flatMap$extension(either.scala:149)
	at polynote.kernel.lang.scal.ScalaSource$$anonfun$completionsAt$1.apply(ScalaSource.scala:320)
	at polynote.kernel.lang.scal.ScalaSource$$anonfun$completionsAt$1.apply(ScalaSource.scala:319)
	at cats.syntax.EitherOps$.flatMap$extension(either.scala:149)
	at polynote.kernel.lang.scal.ScalaSource.completionsAt(ScalaSource.scala:319)
	at polynote.kernel.lang.scal.ScalaInterpreter.completionsAt(ScalaInterpreter.scala:223)
	at polynote.kernel.lang.scal.ScalaInterpreter.completionsAt(ScalaInterpreter.scala:23)
	at polynote.kernel.PolyKernel$$anonfun$completionsAt$1.apply(PolyKernel.scala:154)
	at polynote.kernel.PolyKernel$$anonfun$completionsAt$1.apply(PolyKernel.scala:152)
	at polynote.kernel.PolyKernel$$anonfun$withKernel$1$$anonfun$apply$17$$anonfun$apply$18.apply(PolyKernel.scala:105)
	at polynote.kernel.PolyKernel$$anonfun$withKernel$1$$anonfun$apply$17$$anonfun$apply$18.apply(PolyKernel.scala:104)
	at cats.effect.IO$Map.apply(IO.scala:1407)
	at cats.effect.IO$Map.apply(IO.scala:1403)
```

I'm not doing anything crazy, just typing out `val x = 100` seems to do it.

**Comment by indig** [comment]
Unfortunately due to #28 these errors cause the SocketSession to hang.

**Comment by jeremyrsmith** [comment]
Fixed by #32 and #34
